src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -2813,5 +2815,15 @@ public boolean isPeerStarted(String peerKey) {
   protected void registerForInterfaceChange(Class<?> clazz) {
     Runtime.getInstance().registerForInterfaceChange(getClass().getCanonicalName(), clazz);
   }
+  
 
+  /**
+   * default emtpy getDefaultConfig - will be overriden by services which provide sane defaults
+   * @return
+   */
+ static public Map<String, ServiceConfig> getDefaultConfig() {
```

Comment:
I thought we were going to change the return signature here to force a LinkedHashMap ?

Reply:
heh .. upcoming commit - we're on the same page

Reply:
This method... should be fixed.. to have the correct return signature and it should be moved to the ServiceInterface.